HelpSource/Classes/Set.schelp

```diff
@@ -41,23 +41,23 @@ Returns the object at the internal strong::index::. This index is not determinis
 subsection::Set specific operations
 
 method::sect, &
-Return the set theoretical intersection of this and strong::that::.
+Return the set theoretical intersection of this and strong::that::. The function will search for strong::equal:: objects occurring in both sets and return a new set containing those.
```

Comment:
for ::sect and ::union I would personally stick to the "equality (not identity)" style you use everywhere else. Also "equal (not identical)" would be fine for me.